src/client/pages/PackagesPage.tsx

```diff
@@ -1,37 +1,36 @@
 import React, { useCallback, useContext, useState } from 'react';
 
 import Banner from 'components/Banner';
-import CardPackage from 'components/card/CardPackage';
-import CreatePackageModal from 'components/modals/CreatePackageModal';
-import DynamicFlash from 'components/DynamicFlash';
-import withModal from 'components/WithModal';
-import UserContext from 'contexts/UserContext';
-import MainLayout from 'layouts/MainLayout';
 import Alert from 'components/base/Alert';
-import CardPackageData from 'datatypes/CardPackage';
 import Button from 'components/base/Button';
-import Spinner from 'components/base/Spinner';
-import { Row, Col, Flexbox } from 'components/base/Layout';
+import Controls from 'components/base/Controls';
 import Input from 'components/base/Input';
+import { Col, Flexbox, Row } from 'components/base/Layout';
+import Link from 'components/base/Link';
+import ResponsiveDiv from 'components/base/ResponsiveDiv';
 import Select from 'components/base/Select';
-import Controls from 'components/base/Controls';
+import Spinner from 'components/base/Spinner';
 import Text from 'components/base/Text';
-import useQueryParam from 'hooks/useQueryParam';
+import CardPackage from 'components/card/CardPackage';
+import DynamicFlash from 'components/DynamicFlash';
+import CreatePackageModal from 'components/modals/CreatePackageModal';
 import RenderToRoot from 'components/RenderToRoot';
-import ResponsiveDiv from 'components/base/ResponsiveDiv';
-import Link from 'components/base/Link';
+import withModal from 'components/WithModal';
 import { CSRFContext } from 'contexts/CSRFContext';
+import UserContext from 'contexts/UserContext';
+import CardPackageData from 'datatypes/CardPackage';
+import useQueryParam from 'hooks/useQueryParam';
+import MainLayout from 'layouts/MainLayout';
 
 const CreatePackageModalLink = withModal(Button, CreatePackageModal);
 
 interface PackagesPageProps {
   loginCallback?: string;
   items: CardPackageData[];
   lastKey: string | null;
-  activePage: 'approved' | 'submitted' | 'user';
```

Comment:
activePage was only used in a dependencies but not the function, and once that was fixed up it was clear the variable isn't used anywhere